ZzaDashboard/ViewModel/MainViewModel.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Input;
+using Zza.Data;
+
+namespace ZzaDashboard.ViewModel
+{
+    class MainViewModel : INotifyPropertyChanged
```

Comment:
Missed access modifier :).
I am trying to require everybody to specify access modifiers explicitly for all members.

Reply:
Name should follow convention.
MainWindowViewModel